DockerfileDebug

```diff
@@ -0,0 +1,21 @@
+
+# NOTES:
+# 1. This Docker file is used to define a ConcordBFT debug image.
+# 2. It extends ./Dockerfile release image and adds debug, performance, and analysis tools for Concord-BFT development.
+# 3. DockerfileDebug does not have an entry point because everything has to be managed outside of it.
+# 2. Please do not install anything directly in this file.
+#    You need to use "install_deps_debug.sh" - we want to keep a possibility to develop w/o docker.
+# 3. If a tool or library that you install in the "install_deps_debug.sh" is needed only for building 3rd party
+#   tools/libraries please uninstall them after the script run - we want to keep image' size minimal.
+#   For example wget and pip-tools.
+
+ARG CONCORD_BFT_DOCKER_IMAGE_FULL_PATH_RELEASE
```

Comment:
A default value for ARG is usually supplied

Reply:
We agreed it is not nesssasary. Thanks, closing.